docs/LwM2M_Client/OpenThread/Adding_device.md

```diff
@@ -0,0 +1,167 @@
+# Adding nRF board to OpenThread
+
+Integrate your Nordic nRF52840-DK board to an active OpenThread Border Router.
+
+## Prerequisites
+
+- An active **OpenThread Border Router**.
+- The **nRF52840-DK** board with a USB cable.
+- Installed **minicom** or **RealTerm** (for Linux or Mac) or **PuTTy** (for Windows) or other serial communication program.
+- Installed **nrfjprog** from [Nordic Semiconductor page](https://www.nordicsemi.com/Products/Development-tools/nrf-command-line-tools/download)
+- A user with access to the [**Coiote IoT Device Management**](https://eu.iot.avsystem.cloud/ui/device/inventory) platform and appropriate permissions.
+
+## Connecting to the OpenThread
+
+Check your active Border Router **OTBR_IP_address** on which your Border Router works. This will be necessary for further steps.
+
+### Connection through Wi-Fi
+
+You have to be attached to the same Wifi network as your OpenThread Border Router. To join your device to Border Router, open any web browser and connect to the WebUI by typing **OTBR_IP_address:80**.
```

Comment:
```suggestion
### Connecting to OpenThread Boarder Router WebUI

You have to be attached to the same network as your OpenThread Border Router. To configure the Border Router open any web browser and connect to the WebUI by typing **OTBR_IP_address:80**.
```

Reply:
We've got ## Connecting to the OpenThread two lines above so I don't think it will be good to repeat the title. MAybe something like ###Conneting to WebUI?